src/components/Tabs/Tabs.module.css

```diff
@@ -5,12 +5,57 @@
 \*------------------------------------*/
 
 /**
- * 1) List of of links where each link toggles open associated information
+ * List of of links where each link toggles open associated information
  */
-/* stylelint-disable-next-line block-no-empty */
 .tabs {
+  position: relative;
 }
 
+/**
+ * Fade scrollable indicators to display if there is scrollable area on either side.
+ * 1) Indicators are non interactable and should be able to click tab under them.
+ */
+.tabs--scrollable-left::before,
+.tabs--scrollable-right::after {
+  pointer-events: none; /* 1 */
+  content: '';
+  position: absolute;
+  top: 0;
+  width: var(--eds-size-8);
+
+  height: 2.75rem;
+  @media all and (max-width: $eds-bp-md) {
+    height: 2.5rem;
+  }
+}
+
+/**
+ * Left fade scrollable indicator.
+ */
+.tabs--scrollable-left::before {
+  left: 0;
+  background-image: linear-gradient(
+    to right,
+    var(--eds-theme-color-background-neutral-default),
+    transparent 90%
+  );
```

Comment:
Another thing you could try is a mask with a linear gradient on the `tabs__list` (or the header). That way, it would fade into the background no matter what the background color is behind the tabs, not just on white backgrounds. https://stackoverflow.com/questions/60996982/css-opacity-gradient-top-to-bottom-without-color (Feel free to ignore this comment or make a separate task if it feels like too much work right now; it's not blocking.)

Reply:
I think using mask is a more valid approach. Will try it, thanks!

Reply:
Definitely looks a lot better, thanks!


https://user-images.githubusercontent.com/86632227/175438883-5eb81d9b-1c4d-4b9e-81c2-dfadc9f0f87f.mov


Now there is a visual bug with the mask covering the scroll bar as well, but I'm going to file a separate [backlog ticket](https://app.shortcut.com/czi-edu/story/202962) for it 

![Screen Shot 2022-06-23 at 4 51 02 PM](https://user-images.githubusercontent.com/86632227/175438996-05aa7366-cec6-49db-ab40-08181f356ec6.png)
![Screen Shot 2022-06-23 at 5 47 57 PM](https://user-images.githubusercontent.com/86632227/175439000-4fd6a057-8549-498e-804b-14b73b1e9636.png)

Reply:
Yup will do!